File: problem_twenty_six.py
import math
# 1st ways
value = int(input("Enter the number:"))
def MyFunction(number):
    lst =[]
    while number > 9:
        mod = number % 10
        lst.append(math.floor(mod))
        div = number/10
        number = div
    lst.append(math.floor(number))    
    lenht = len(lst)
    count = 0
    for i in lst:
        count+= i**lenht
    return count
if value == MyFunction(value):
    print(value,"is an armstrong number!")
else:
    print(value, "is not an armstrong numbr!")       


# 2 nd ways


value = int(input("Enter the number:"))
def Armstrong_number(value):
    lst =list(str(value))
    count = 0
    for i in lst:
        count += int(i)**len(lst)
    return count   
if value == Armstrong_number(value):
    print(value,"is an armstrong number:")
else:
    print(value,"is not an armstrong number:")

File: problem_twenty_six.js
// 1 st ways
var number = 371
function Myfunction(number){
    var lst =[]
    while (number > 9){
        var mod = number % 10
        lst.push(Math.floor(mod))
        var div = number/ 10
        number = div
    }
    lst.push(Math.floor(number))
    var len = lst.length
    var count = 0
    for(var i =0; lst.length>i;i++){
        count += lst[i]**len
    }
    return count
}
if (number == Myfunction(number)){
    console.log(number,"is an armstrong number!")

}else{
    console.log(number,"is not an armstrong number!")
}



//  2nd ways
var number = 311
function arMsTrong(number){
   var num= number.toString().split("")
   var len = num.length
   var count = 0
   for(var i = 0; len> i;i++){
      count+= (num[i]**len)
   }
   return count
}
if(number == arMsTrong(number)){
    console.log(number,"is an armstron number !")
}else{
    console.log(number,'is not an armstrong number !')
}






